lua/barbar/ui/render.lua

```diff
@@ -29,10 +30,10 @@ local animate = require'barbar.animate'
 local Buffer = require'barbar.buffer'
 local config = require'barbar.config'
 -- local fs = require'barbar.fs' -- For debugging purposes
-local groups = require'barbar.groups'
+local Nodes = require('barbar.ui.nodes')
```

Comment:
lint: either change none or all of them, otherwise the local inconsistence is too strong. let's leave the requires wihout parens for now.